packages/kbn-eslint-plugin-packages/src/rules/no_naked_eslint_disable.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import Eslint from 'eslint';
+
+const NAKED_DISABLE_MSG_ID = 'no-naked-eslint-disable';
+const messages = {
+  [NAKED_DISABLE_MSG_ID]:
+    'Using a naked eslint disable is not allowed. Please specify the specific rules to disable.',
+};
+
+const meta: Eslint.Rule.RuleMetaData = {
+  type: 'problem',
+  fixable: 'code',
+  docs: {
+    description:
+      'Prevents declaring naked eslint-disable* comments who do not provide specific rules to disable',
+  },
+  messages,
+};
+
+const nakedEslintDisableRegex =
```

Comment:
nit: I'd probably name this `ESLINT_DISABLE_RE`, it matches non-naked rules too.

Reply:
👍